examples/semantic_segmentation/semantic_segmentation/evaluator.py

```diff
@@ -38,31 +36,13 @@
 from kolena.workflow import EvaluationResults
 from kolena.workflow import Plot
 from kolena.workflow import TestCases
-from kolena.workflow.annotation import SegmentationMask
 from kolena.workflow.metrics import f1_score as compute_f1_score
 from kolena.workflow.metrics import precision as compute_precision
 from kolena.workflow.metrics import recall as compute_recall
 from kolena.workflow.plot import Curve
 from kolena.workflow.plot import CurvePlot
 
 
-def _upload_sample_result_masks(
```

Comment:
Is this not called?

Reply:
no it was moved to data_loader.py but didn't get removed.